.github/workflows/Build.yml

```diff
@@ -24,11 +24,15 @@ jobs:
     - name: Install requirements
       run: |
         sudo apt-get update
-        sudo apt-get install -y gfortran openmpi-bin libopenmpi-dev
+        sudo apt-get install -y gfortran openmpi-bin libopenmpi-dev cmake
 
     # Runs a set of commands using the runners shell
     - name: Compile
-      run: make BUILD=dev
+      run: |
+        FC=mpif90 cmake -B build -DCMAKE_BUILD_TYPE=dev
+        make -C build decomp2d
```

Comment:
Should be cmake ?

Reply:
Yes, we should follow the style from the README. Will fix

Reply:
Addressed by 75bd117